mev-relay-rs/src/relay.rs

```diff
@@ -618,19 +602,18 @@ impl BlindedBlockRelayer for Relay {
 
             self.validate_builder_submission_trusted(bid_trace, signed_submission.payload())?;
             debug!(%auction_request, "validated builder submission");
-            (auction_request, bid_trace.value, bid_trace.builder_public_key.clone())
```

Comment:
altho seems a bit nicer to just say `builder_public_key.clone()` here instead of going thru bid trace